src/FilterChain.php

```diff
@@ -20,14 +20,17 @@
 use function sprintf;
 use function strtolower;
 
+/**
+ * @final
```

Comment:
Not sure if we can add this here, unless we plan a migration to actual keyword in the next major?

Reply:
This was an assumption that final by default would be a goal for any future major - Shall I revert?

Reply:
If we do it in the next major, we're good here :+1: